hedera-node/hedera-app/build.gradle.kts

```
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id("com.hedera.hashgraph.hedera-conventions")
+}
+
+description = "Hedera Application - Implementation"
+
+dependencies {
+    project(":hedera-node:hedera-app-api")
```

The original code creates an intra-project dependency which spans all source sets with compile, runtime, and transitive dependency scopes. We should be more explicit as to which source sets and what scopes to which this dependency applies.

```suggestion
    api(project(":hedera-node:hedera-app-api"))
```